### ISSUE TITLE
I'm getting this error!

### ISSUE THREAD

**Issue by Chartone** [BeepIsla/CSGO-Overwatch-Bot#69]
```
Requested Overwatch case
Unexpected behaviour: Got a completion without sending one. Retrying in 30 seconds
--------------------
Requested Overwatch case
Unexpected behaviour: Got a completion without sending one. Retrying in 30 seconds
--------------------
Requested Overwatch case
Downloading case 3430206786643165401 from url http://replay191.valve.net/730/003430080248316690781_1408446636.dem.bz2
--------------------
Requested Overwatch case
Downloading case 3430206786643165401 from url http://replay191.valve.net/730/003430080248316690781_1408446636.dem.bz2
Finished downloading 3430206786643165401, unpacking...

internal/streams/legacy.js:61
      throw er; // Unhandled stream error in pipe.
      ^
Error
    at new Bzip2Error (D:\Docs\CSGO\CSGO Overwatch\node_modules\unbzip2-stream\lib\bzip2.js:31:19)
    at Object.Error (D:\Docs\CSGO\CSGO Overwatch\node_modules\unbzip2-stream\lib\bzip2.js:36:37)
    at Object.bzip2.decompress (D:\Docs\CSGO\CSGO Overwatch\node_modules\unbzip2-stream\lib\bzip2.js:300:46)
    at decompressBlock (D:\Docs\CSGO\CSGO Overwatch\node_modules\unbzip2-stream\index.js:30:29)
    at decompressAndQueue (D:\Docs\CSGO\CSGO Overwatch\node_modules\unbzip2-stream\index.js:47:20)
    at Stream.write (D:\Docs\CSGO\CSGO Overwatch\node_modules\unbzip2-stream\index.js:76:17)
    at Stream.stream.write (D:\Docs\CSGO\CSGO Overwatch\node_modules\through\index.js:26:11)
    at ReadStream.ondata (_stream_readable.js:716:22)
    at ReadStream.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:271:9)
    at ReadStream.Readable.push (_stream_readable.js:212:10)
    at internal/fs/streams.js:215:14
    at FSReqCallback.wrapper [as oncomplete] (fs.js:520:5) {
  message: 'Boom.'
}
```

**Comment by BeepIsla** [comment]
Why is it requesting an OW case twice?

**Comment by Chartone** [comment]
As far as I can tell, he tried it once, but he couldn't and after 30s he tried again!

**Comment by BeepIsla** [comment]
It would say something if it fails and tries again, it doesn't do that here, it just downloads it twice. If you just restart, does it still happen?

**Comment by Chartone** [comment]
No, I think there was a drop in my connection! Hence this error.

**Comment by MaRk5man5** [comment]
I Got The Same Error Today.

**Comment by BeepIsla** [comment]
Released version 2.0 - This might be fixed or not, re-open if not fixed.
